searchcore/src/vespa/searchcore/proton/attribute/flushableattribute.cpp

```diff
@@ -62,7 +62,7 @@ FlushableAttribute::Flusher::Flusher(FlushableAttribute & fattr, SerialNum syncT
       _syncToken(syncToken),
       _flushFile("")
 {
-    fattr._attr->commit(syncToken, syncToken);
+    fattr._attr->commit(CommitParam(syncToken, true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour.